Vic2ToHoI4/Source/V2World/Military/ArmyFactory.cpp

```diff
@@ -7,6 +7,12 @@
 
 Vic2::Army::Factory::Factory()
 {
+	registerKeyword("id", [this](std::istream& theStream) {
+		const auto& idStr = commonItems::stringOfItem{theStream}.getString();
+		std::smatch sm;
+		std::regex_search(idStr, sm, std::regex("id=([0-9]+)"));
```

Comment:
Does this impact performance? Annoying as it'd be, maybe it'd be best to create a little factory class just for those ids.